install/kubernetes/cilium/templates/cilium-nodeinit-daemonset.yaml

```
@@ -110,7 +110,14 @@ spec:
 
 {{- if .Values.nodeinit.reconfigureKubelet }}
                     echo "Changing kubelet configuration to --network-plugin=kubenet"
-                    sed -i "s:--network-plugin=cni\ --cni-bin-dir={{ .Values.cni.binPath }}:--network-plugin=kubenet:g" /etc/default/kubelet
+                    if [[ -f /etc/default/kubelet ]];
+                    then
```

nit: for consistency's sake
```suggestion
                    if [[ -f /etc/default/kubelet ]]; then
```
Same for the other conditions.